bad_code.py

```diff
@@ -0,0 +1,74 @@
+import re
+import datetime
+from collections import OrderedDict
+
+def create_racer_abbreviations_dict(file_name):
+    """Retrieves {'abbreviation': (name, team)}" format dict from abbreviations.txt"""
+    abbreviations_dict = {}
+    with open(file_name, 'r') as fn:
+        for line in fn:
+            matchObj = re.match(r'^(\w+)_([a-zA-Z\s]+)_([a-zA-Z\s]+)$', line)
+            # group(1) is abbreviation, i.e 'SVM'
+            abbreviations_dict[matchObj.group(1)] = (
+                matchObj.group(2),  # name of a pilot
+                matchObj.group(3).rstrip(),  # team
+            )
```

Comment:
Instead of these comments could be simplified to:
```suggestion
            # group(1) is abbreviation, i.e 'SVM'
            abbreviation, pilot_name, team_name = group(1), group(2), group(3)
            abbreviations_dict[abbreviation] = (pilot_name, team_name)
```